jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletHandler.java

```diff
@@ -1478,7 +1493,16 @@ public ServletPathMapping getServletPathMapping(String pathInContext)
             if (_servletPathMapping != null)
                 return _servletPathMapping;
             if (_pathSpec != null)
-                return new ServletPathMapping(_pathSpec, _servletHolder.getName(), pathInContext);
+                return new ServletPathMapping(_pathSpec, _servletHolder.getName(), pathInContext, null);
+            return null;
+        }
+
+        public ServletPathMapping getServletPathMapping(String pathInContext, MatchedPath matchedPath)
```

Comment:
@gregw this is the new method that isn't being used consistently that I'm concerned about.

There is _also_ a `getServletPathMapping(String pathInContext)` above this method that I couldn't get rid of (we should be able to get rid of it)